src/libs/actions/Policy.ts

```diff
@@ -124,7 +125,7 @@ type OptimisticCustomUnits = {
     outputCurrency: string;
 };
 
-type PoliciesRecord = Record<string, OnyxEntry<Policy>>;
+type PoliciesRecord = Record<string, OnyxEntry<PolicySelector>>;
```

Comment:
```suggestion
type PoliciesRecord = Record<string, OnyxEntry<Policy>>;
```

Let's revert this unnecessary change